pwiz_tools/Skyline/TestTutorial/ExistingExperimentsTutorialTest.cs

```diff
@@ -641,6 +651,22 @@ private static void NormalizeGraphToHeavy()
             SkylineWindow.UpdatePeakAreaGraph();
         }
 
+        private void VerifyRdotPLabels(string[] replicates, string[] rdotps)
+        {
+            if (!Program.SkylineOffscreen)
+            {
+                var rdotpLabels = SkylineWindow.GraphPeakArea.GraphControl.GraphPane.GraphObjList.OfType<TextObj>().ToList()
+                    .FindAll(txt => txt.Text.StartsWith(@"rdotp")).Select((obj) => obj.Text).ToArray();
+
+                Assert.IsTrue(Enumerable.Range(0, replicates.Length).All(i =>

```

Comment:
It would be better if you could make this into a for loop which loops over all of the replicates, and calls Assert for each replicate.
It's really hard to debug something like this when it fails because all you get it "Assert.IsTrue failed" and there's no indication of which replicate did it, and what the value was that was being tested.